cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/io/cdap/cdap/etl/api/engine/sql/StandardSQLCapabilities.java

```diff
@@ -19,15 +19,23 @@
 import io.cdap.cdap.etl.api.relational.Capability;
 
 /**
- * Defines capabilities for SQL Engine factories
+ * Defines capabilities for SQL Engine factories.
  */
 public enum StandardSQLCapabilities implements Capability {
```

Comment:
Note: We can't change this, so please use `@SuppressWarnings({"abbreviationAsWordInName"})`, see https://checkstyle.sourceforge.io/apidocs/com/puppycrawl/tools/checkstyle/filters/SuppressWarningsFilter.html